pkg/web/metricsHandler_test.go

```diff
@@ -0,0 +1,140 @@
+package web
+
+import (
+	"testing"
+
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"github.com/aerogear/aerogear-metrics-api/pkg/mobile"
+	"bytes"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/assert"
```

Comment:
@darahayes I did a `go get` to get Testify locally and use here. however, I am not sure if I should add this dependency `dep` as it is a testing dependency only. how to handle this in Go world?

Reply:
hey, wait a minute! this thing is compiled to native code so there is only compile time dependencies! ignore my question @darahayes . I will just add new deps to dep tool.